frontend/src/components/Context.tsx

```diff
@@ -0,0 +1,26 @@
+import React, {
+  createContext,
+  PropsWithChildren,
+  useEffect,
+  useState,
+} from "react";
+import Axios, { AxiosResponse } from "axios";
+import { UserInterface } from "../utils/Interfaces";
+
+export const myContext = createContext<Partial<UserInterface>>({});
+
+export default function Context(props: PropsWithChildren<any>) {
+  const [user, setUser] = useState<UserInterface>();
+
+  useEffect(() => {
+    Axios.get("http://localhost:5050/user", { withCredentials: true }).then(
```

Comment:
Axios instance is wondering why you have created it but not using it 😆

Reply:
i was lazy to create new instances for the complaint and user routes...